build/archives/BUILD.gn

```diff
@@ -16,6 +16,7 @@ generated_file("artifacts_entitlement_config") {
       "//flutter/impeller/compiler:impellerc",
       "//flutter/impeller/tessellator:tessellator_shared",
       "//flutter/shell/testing:testing",
+      "//flutter/tools/path_ops:path_ops",
```

Comment:
Why do we need to add this dep?

Reply:
yeah this dependency is needed to collect the metadata from tools/path_ops/BUILD.gn.

we defined a data_keys field in this target. So that if a matching data key is present in tools/path_ops/BUILD.gn, it is propagated and collected into the meta data file.